app/furniture/journal/journals/_journal.html.erb

```diff
@@ -1,4 +1,10 @@
 <section class="max-w-3xl mx-auto">
+  <%- if policy(new_entry).new? %>
+    <div class="text-center p-2">
+      <%= link_to "New Journal Entry", journal.location(:new, child: :entry) %>
+    </div>
+  <%- end %>
+
```

Comment:
I had to code this blind. 😓 

I can't get to the page for creating new journal entries on my local. I added a Journal Gizmo to a room but can't figure out where to go to add an entry or look at journal entry lists etc. 

Is our UI that bad or did I forget to set up something?
<img width="1735" alt="Screenshot 2023-06-20 at 7 45 40 PM" src="https://github.com/zinc-collective/convene/assets/16140873/5d003841-b725-4951-836e-3e122175ba7f">

I keep clicking the room name of the room I added the Journal gizmo to, but nothing happens. 
<img width="776" alt="Screenshot 2023-06-20 at 7 46 13 PM" src="https://github.com/zinc-collective/convene/assets/16140873/5fa9d6e4-397f-4a9d-a45e-f7288db25ce6">

And I tried manually crafting the correct URL path based off the route syntax. No dice 😞 

`/spaces/:space_id/rooms/:room_id/journals/:journal_id/entries/new(.:format)`


`http://localhost:3000/spaces/test/rooms/food-room/journals/1/entries/new`

Reply:
I wonder what would happen if you put a bit of content inside of the `if` and outside the `if` in order to identify whether the `if policy(entry).new?` was why it was being hidden or if it was a different issue.

My guess is that the `Journal::EntryPolicy` expects the `current_person`  to be a `member?`  of the `current_space`; and it doesn't care that you are an `operator?`.

If that does seem like the case, LMK and I'll create an enhancement issue (or you can create one?) so we don't lose track!